test/docker/test_readiness.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+check_readiness() {
+   local response=$(curl -s "${GITLAB_URL}")
```

Comment:
Shouldn't the URL be `/-/readiness`?

Reply:
Yea, I just put some template url there for now as i debug my syntax errors